pkg/manifests/vllm/deployment.yaml

```diff
@@ -39,7 +39,7 @@ spec:
           - "8000"
           - "--enable-lora"
           - "--max-loras"
-          - "4"
+          - "2"
```

Comment:
Makes it easier to work with smaller gpus, also our guides are focused on two adapters now.